pkg/apiserver/user/verify_sql_user.go

```diff
@@ -59,14 +60,20 @@ func VerifySQLUser(tidbClient *tidb.Client, userName, password string) (writeabl
 	if err != nil {
 		return false, err
 	}
-	// 2. Get grants
+	// 2. Check SkipGrantTable
+	// Note: Currently, if TiDB enable the skip-grant-table, running `show grants` will get error.
+	// So this is a workaround before the above bug is fixed.
+	if config.Security.SkipGrantTable {
+		return true, nil
```

Comment:
In this situation, should the user has the write privilege?

Reply:
I think the user can have the write privilege.